.circleci/config.yml

```diff
@@ -486,16 +486,17 @@ jobs:
           name: Docker build
           command: |
             ROUTER_TAG=ghcr.io/apollographql/router
-            # Build debug image
-            docker build --build-arg ROUTER_RELEASE=${VERSION} -f dockerfiles/Dockerfile.router -t ${ROUTER_TAG}:${VERSION}-debug .
-            # Build release image
-            docker build --build-arg ROUTER_RELEASE=${VERSION} -f dockerfiles/Dockerfile.router -t ${ROUTER_TAG}:${VERSION} .
+            # Create a multi-arch builder which works properly under qemu
+            docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+            docker context create buildx-build
+            docker buildx create --driver docker-container --use buildx-build
+            docker buildx inspect --bootstrap
```

Comment:
What does this line do?  Is it just output?

Reply:
It is. I could remove it, but it contains useful debugging information if we have a proble.

Reply:
works for me!